corona_stats/packages/corona_stats/corona_stats.js

```diff
@@ -226,7 +251,7 @@ const success = (header, fields, footer) => {
   for (const property in fields) {
     response.blocks[1].fields.push({
       type: 'mrkdwn',
-      text: `${property}:   *${(fields[property] || 0)}*`,
+      text: `${property}:  ${fields[property]?`*${fields[property]}*`:''}`,
```

Comment:
Is it possible to check for property before we push? Something like: 
```js
If (fields [property]) {
response.
}
```

Sorry for the formatting, I'm on mobile.

Reply:
sure, thanks.
the check can be promoted, but we'll lose the property name as well.
i.e. I want to show New Cases, even if not new cases reported today, so that it catches attention. Earlier we're showing 0 instead of blank.

We can revert back to 0, if the blank doesn't make sense.

Reply:
Okay, I got it. 0 might be more descriptive but both are okay I guess.

Reply:
Earlier we're showing 0, but that gives a sense of certainty but the original source is showing blank only. Still, I am not sure what it should be. Maybe @niks3089 will help building consensus.

Reply:
reverted to `0` as it was well accepted.